zcash_client_sqlite/src/wallet.rs

```diff
@@ -1167,47 +1167,12 @@ pub(crate) fn get_wallet_summary<P: consensus::Parameters>(
     drop(sapling_trace);
 
     #[cfg(feature = "transparent-inputs")]
-    {
-        let transparent_trace = tracing::info_span!("stmt_transparent_balances").entered();
```

Comment:
Preserve this trace by adding a `#[tracing::instrument]` annotation to the new `transparent::add_transparent_account_balances` method.